tests/functional/instance_manager/__init__.py

```diff
@@ -0,0 +1,122 @@
+# coding=utf-8
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+"""Setup for the package tests."""
+
+import copy
+import json
+import multiprocessing
+import os
+import shlex
+import shutil
+import subprocess
+import sys
+import time
+import uuid
+from subprocess import CalledProcessError
+
+from libtest import get_free_port
+from libtest import project
+from libtest import codechecker
+from libtest import env
+
+# Stopping events for CodeChecker servers.
+__SERVER_EVENTS = [multiprocessing.Event(),
```

Comment:
Maybe we could use separate variable names like `server1_stop` for each stop event. 
Which make the stopping and starting more explicit and harder to mix the events.
`start_server(codechecker_1, test_cfg, server1_stop)` and to stop `set_event(server1_stop)`